### PR TITLE
Add a visible page border in HCM (bug 1776914)

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

**Comment by MReschenberg** [comment]
😁 thank you!

**Review by ahmed82 — APPROVED** [review]
Looks good

File: web/pdf_viewer.css [hunk 1]
```diff
@@ -30,7 +31,8 @@
   :root {
     --pdfViewer-padding-bottom: 9px;
     --page-margin: 9px auto 0;
```
Comment (Snuffleupagus) [inline web/pdf_viewer.css:33]: *I've not tested, but simply reading the surrounding the code:*
Don't you need to change these values, to account for the fact that there's now a border added just below?
Comparing with the "normal" rules, should this be updated to e.g. the following now?
```suggestion
    --page-margin: 8px auto -1px;
```

File: web/pdf_viewer.css [hunk 2]
```diff
@@ -30,7 +31,8 @@
...
-    --page-border: none;
+    --page-border: 1px solid CanvasText;
+    --page-border-image: none;
     --spreadHorizontalWrapped-margin-LR: 4.5px;
```
Comment (Snuffleupagus) [inline web/pdf_viewer.css:36]: *I've not tested, but simply reading the surrounding the code:*
Don't you need to change this values, to account for the fact that there's now a border added just above?
Comparing with the "normal" rules, should this be updated to e.g. the following now?
```suggestion
    --spreadHorizontalWrapped-margin-LR: 3.5px;
```

**Review by Snuffleupagus — APPROVED** [review]
r=me, thank you!

**Pull request merged**